src/streaming/controllers/MediaController.js

```diff
@@ -72,6 +85,43 @@ MediaPlayer.dependencies.MediaController = function () {
              };
          },
 
+         selectInitialTrack = function(tracks) {
+             var initTrack,
+                 max = 0,
+                 tmp,
+                 idx = 0,
+                 mode = this.getSelectionModeForInitialTrack();
+
```

Comment:
I think ideally this would be able to combine the two.   Where there's two identical "highest bitrate" it still takes the first in the list, and equally two with similar range it takes the first.   Would be nice to have a combination of selection types.